src/components/ViewMenu.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FunctionComponent } from "react";
+import Button from "@material-ui/core/Button";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import ViewWeekIcon from "@material-ui/icons/ViewWeek";
+import ViewDayIcon from "@material-ui/icons/ViewDay";
+import ViewComfyIcon from "@material-ui/icons/ViewComfy";
+import ViewAgendaIcon from "@material-ui/icons/ViewAgenda";
+import {
+  CalendarAction,
+  CalendarView,
```

Comment:
this should be deletable after proposed changes

Reply:
You mean CalendarView can be deleted, right? the others are needed as configured, at least how i'm looking at it.

Reply:
Yes, that was meant to just be on line # 11